contributing/releasing.md

```diff
@@ -91,6 +95,13 @@ Inspect the API changes to get a quick sense of whether there might be an API-br
 1. Otherwise, if any public APIs added, then this release is a *minor* release.
 1. Otherwise, this release *might* still be a bug fix release.
 
+### New Features / Top Level Description 
+
+In the "New Features" sub-category generated by the API diff tool, please provide more detail as to what is new in the release, also providing examples on how to use the added public API properties/methods.
```

Comment:
Can you give an example of what you expect "providing examples" to look like?

Reply:
done